src/app/shared/components/support-information/support-information.component.html

```diff
@@ -0,0 +1,14 @@
+<ng-container *ngIf="isShow">
+  <div class="support-information">
+    <h4>{{ 'ERRORS.SUPPORT_INFORMATION' | translate }}</h4>
+    <button
```

Comment:
Why do not you use <mat-expansion-panel>? (https://material.angular.io/components/expansion/overview)
I suppose that it looks better, and we do not need any CSS for it. It is just a comment, you can use your own solution :)